TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/ObstacleCoursePath.java

```diff
@@ -8,16 +8,25 @@
 import org.firstinspires.ftc.teamcode.drive.SampleMecanumDrive;
 import org.firstinspires.ftc.teamcode.trajectorysequence.TrajectorySequence;
 
-
 @Autonomous (group = "drive")
+
 public class ObstacleCoursePath extends LinearOpMode {
+
     @Override
     public void runOpMode() {
+        SampleMecanumDrive drive = new SampleMecanumDrive(hardwareMap);
 
+        drive.setPoseEstimate(new Pose2d(-60, 60, Math.toRadians(270)));
 
-        SampleMecanumDrive drive = new SampleMecanumDrive(hardwareMap);
+        Trajectory trajectoryFirstPart = drive.trajectoryBuilder(new Pose2d(-60, 60, Math.toRadians(270)))
+                .splineTo(new Vector2d(-57.3, 29), Math.toRadians(270))
+                .build();
+
+        Trajectory trajectorySecondPart = drive.trajectoryBuilder(trajectoryFirstPart.end())
+                .splineTo(new Vector2d(7.5, 12), Math.toRadians(0))
```

Comment:
Kevina's code (2nd Chunk) starts at (5, 10) while yours ends at (7.5, 12). This should be fixed.